data/entities.json5

```diff
@@ -11347,5 +11347,53 @@
     "categories": ["advertising"],
     "domains": ["*.clickguardian.app", "*.clickguardian.co.uk"],
     "examples": ["v2.clickguardian.app", "protection.clickguardian.co.uk"]
-  }
+  },
+  {
+    "name": "Smartsupp",
+    "company": "Smartsupp.com",
+    "homepage": "https://www.smartsupp.com",
+    "categories": ["customer-success"],
+    "domains": [
+      "*.smartsuppchat.com",
+      "*.smartsupp.com",
+      "smartsupp-widget-161959.c.cdn77.org",
+      "smartsupp-files-*.cdn77.org"
```

Comment:
is the number the same for this `-files` domain? can't support the wildcard unfortunately :/

if we don't know it, could just leave it in examples for now?

Reply:
```suggestion:-1
```

Reply:
I've changed it in the last commit.